miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/QualityControlService.java

```diff
@@ -0,0 +1,35 @@
+package uk.ac.bbsrc.tgac.miso.service;
+
+import java.io.IOException;
+import java.util.Collection;
+import java.util.stream.Collectors;
+
+import uk.ac.bbsrc.tgac.miso.core.data.QC;
+import uk.ac.bbsrc.tgac.miso.core.data.QcTarget;
+import uk.ac.bbsrc.tgac.miso.core.data.QualityControllable;
+import uk.ac.bbsrc.tgac.miso.core.data.type.QcType;
+
+public interface QualityControlService {
+
+  QC createQC(QC qc) throws IOException;
+
+  QC get(QcTarget target, Long id) throws IOException;
+
+  QualityControllable<?> getEntity(QcTarget target, long ownerId) throws IOException;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove usage of generic wildcard type. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1452)